lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -935,9 +969,12 @@ def corrupt_agw_state(self, key: str):
 
     def restart_all_services(self):
         """Restart all magma services on magma_dev VM"""
-        self.exec_command(
-            "sudo service magma@* stop ; sudo service magma@magmad start",
-        )
+        if self._init_system == InitMode.SYSTEMD:
+            self.exec_command(
+                "sudo service magma@* stop ; sudo service magma@magmad start",
```

Comment:
Hi @sebathomas here it looks good as all services have restarted. Do we need similar handling for test case where an individual service have started. Say for test case, test_sctp_shutdown_while_mme_is_stopped.py mme, mobilityd, pipelined services have stopped and started mme service.
Please find the test case here, https://github.com/magma/magma/blob/master/lte/gateway/python/integ_tests/s1aptests/test_sctp_shutdown_while_mme_is_stopped.py

Reply:
Thanks for looking into this. Yes, I expect more changes will be necessary to run all test, this will be handled in the follow-up ticket #13684.